Examples/Framework/include/ActsExamples/EventData/ScalingCalibrator.hpp

```diff
@@ -0,0 +1,63 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include <Acts/Geometry/GeometryIdentifier.hpp>
+#include <ActsExamples/EventData/MeasurementCalibration.hpp>
+
+#include <TFile.h>
+#include <TH2D.h>
+
+namespace ActsExamples {
+
+class ScalingCalibrator : public MeasurementCalibrator {
+ public:
+  struct ConstantTuple {
+    double x_offset;
```

Comment:
Not sure if clang-tidy will also warn you, but we should initialize these values (to 0?) to ensure we have always initialized values here.

Reply:
Done -- initialized the offsets to 0 and the scales to 1.